src/ucs/async/async.c

```diff
@@ -638,6 +638,10 @@ void __ucs_async_poll_missed(ucs_async_context_t *async)
             break;
         }
 
+        if (value == UCS_MPMC_INVALID_VALUE) {
```

Comment:
instead of exposing UCS_MPMC_INVALID_VALUE from mpmc.h, can we update ucs_mpmc_queue_is_empty and ucs_mpmc_queue_pull to ignore this value?

Reply:
Do we need check it in `ucs_mpmc_queue_is_empty`, too?
This will add lock/unlock inside `ucs_mpmc_queue_is_empty`.
Maybe we can do this check only in `ucs_mpmc_queue_pull`, since it will return `UCS_ERR_NO_PROGRESS` if queue is empty, WDYT?

Reply:
ok